usaspending_api/transactions/delta_models/transaction_search.py

```diff
@@ -389,15 +391,16 @@
 TRANSACTION_SEARCH_POSTGRES_COLUMNS = {k: v["postgres"] for k, v in TRANSACTION_SEARCH_COLUMNS.items() if not v["gold"]}
 TRANSACTION_SEARCH_POSTGRES_GOLD_COLUMNS = {k: v["postgres"] for k, v in TRANSACTION_SEARCH_COLUMNS.items()}
 
-transaction_search_create_sql_string = fr"""
+transaction_search_create_sql_string = rf"""
     CREATE OR REPLACE TABLE {{DESTINATION_TABLE}} (
         {", ".join([f'{key} {val}' for key, val in TRANSACTION_SEARCH_GOLD_DELTA_COLUMNS.items()])}
     )
     USING DELTA
     LOCATION 's3a://{{SPARK_S3_BUCKET}}/{{DELTA_LAKE_S3_PATH}}/{{DESTINATION_DATABASE}}/{{DESTINATION_TABLE}}'
 """
 
-transaction_search_load_sql_string = fr"""
+# TODO: include derivations for recipient_location_congressional_code_current and pop_congressional_code_current
```

Comment:
Not sure on the status of the TODO comment, because I didn't add it. It looks like that comment just moved up a line after formatting.

I think that comment relates to things that @dpb-bah has worked on, so he might be able to comment on the comment? I could be wrong on that though.

Reply:
That TODO should definitely be removed.

Reply:
Where are the commits that introduced this TODO? Removing the TODO should be handled under that branch and story since Andrew did not add this TODO. Although, it might not be possible if the branch is long gone.

Reply:
The TODO should have been removed as part of PR #3816 merged yesterday, but I'm not sure why my branch still has things removed in #3816 but also things introduced in #3816

Reply:
Hmm, have you merged the latest version of QAT into your branch yet?

Reply:
Did you have to resolve merge conflicts at any point? Perhaps, resolving a merge conflict caused these changes to be retained when they shouldn't have

Reply:
I don't remember any merge conflicts, but the file should be cleaner now. I copied the contents of the file from QAT and then re-added my changes on top of the clean file.